### PR TITLE
[onert-micro] This PR adds makefile example for TizenRT

### PR DESCRIPTION
**Pull request by Torrero**
This PR adds makefile example for TizenRT

Makefile was added for the [issue](https://github.com/Samsung/ONE/issues/14184)

ONE-DCO-1.0-Signed-off-by:  Evgenii Maltsev e.maltsev@samsung.com

### CODE REVIEW

File: onert-micro/examples/TizenRT/Makefile [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+CXXFLAGS += -I$(SCHEMA_DIR) -I$(ONERTMICRO_INCLUDE_DIR) -I$(ONERTMICRO_SRC_DIR) -I$(FLATBUFFER_DIR)
+CXXFLAGS += -I$(ONERTMICRO_PAL_COMMON_DIR)
+
+ifeq ($(CONFIG_EXTERNAL_CMSIS_NN), y)
+CXXFLAGS += -I$(ONERTMICRO_PAL_CMSISNN_DIR)
+else
+CXXFLAGS += -I$(ONERTMICRO_PAL_MCU_DIR)
+endif
+ 
+
```
Comment (stamalakhov) [inline onert-micro/examples/TizenRT/Makefile:56]: Is it important to use additional line here?

**Review by stamalakhov — COMMENTED** [review]
Except for minor comment the draft seems to be LGTM!

**Comment by Torrero** [comment]
@chunseoklee PTAL

**Review by chunseoklee — DISMISSED** [review]

File: onert-micro/examples/TizenRT/Makefile (hunk 1 above)
Comment (Torrero) [reply onert-micro/examples/TizenRT/Makefile:55]: changed, thank you!

**Review by stamalakhov — APPROVED** [review]
LGTM! Thank you!

**Review by SlavikMIPT — APPROVED** [review]
LGTM

**Pull request merged**